### PR TITLE
feat(famedly_sync): add role

### PR DESCRIPTION
**Pull request by lukaslihotzki-f**

### CODE REVIEW

File: roles/famedly_sync/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    name: famedly-sync
+    image: "{{ famedly_sync_container_image_reference }}"
+    state: "present"
+    volumes: "{{ famedly_sync_docker_volumes }}"
+    labels: "{{ famedly_sync_docker_labels | default(omit, True) }}"
+    networks: "{{ famedly_sync_docker_networks }}"
+
+- name: "Deploy config"
+  copy:
+    content: "{{ famedly_sync_config | to_nice_yaml }}"
```
Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:29]: Could you give this var a sensible default or if that doesn't make sense assert that it is defined in the beginning and add it to the README?

Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:29]: Please use `to_nice_yaml(indent=2)`

File: roles/famedly_sync/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+
+- name: "Create volume path"
+  file:
+    path: "{{ item }}"
+    state: "directory"
+    mode: "0700"
```
Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:7]: I usually prefer `0755` for the base directory so listing is allowed for everyone.

File: roles/famedly_sync/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  community.docker.docker_container:
+    name: famedly-sync
+    image: "{{ famedly_sync_container_image_reference }}"
+    state: "present"
+    volumes: "{{ famedly_sync_docker_volumes }}"
+    labels: "{{ famedly_sync_docker_labels | default(omit, True) }}"
+    networks: "{{ famedly_sync_docker_networks }}"
+
+- name: "Deploy config"
+  copy:
```
Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:28]: `ansible.builtin.copy`

File: roles/famedly_sync/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+
+- name: "Create volume path"
+  file:
```
Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:4]: `ansible.builtin.file`

File: roles/famedly_sync/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+famedly_sync_path: /opt/famedly-sync
+
+famedly_sync_container_image_force_pull: false
```
Comment (lrsksr) [inline roles/famedly_sync/defaults/main.yml:3]: `famedly_sync_container_image_force_pull: "{{ famedly_sync_container_image_tag is defined }}"`

File: roles/famedly_sync/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    networks: "{{ famedly_sync_docker_networks }}"
+
+- name: "Deploy config"
+  copy:
+    content: "{{ famedly_sync_config | to_nice_yaml }}"
+    dest: "{{ famedly_sync_path }}/config.yaml"
+    mode: "0400"
+
+- name: "Install systemd units"
+  ansible.builtin.template:
```
Comment (lrsksr) [inline roles/famedly_sync/tasks/main.yml:34]: Why not use copy and a `files` directory instead? Are there plans to add Jinja templates to the systemd files?

**Review by lrsksr — CHANGES_REQUESTED** [review]

File: roles/famedly_sync/tasks/main.yml (hunk 2 above)
Comment (lukaslihotzki-f) [reply roles/famedly_sync/tasks/main.yml:7]: Then, everyone would be able to read the "cache" (user database) that is created by famedly-sync on the first run.

File: roles/famedly_sync/tasks/main.yml (hunk 6 above)
Comment (lukaslihotzki-f) [reply roles/famedly_sync/tasks/main.yml:39]: This may make sense for the interval, for example.

**Review by lrsksr — APPROVED** [review]

**Review by lrsksr — APPROVED** [review]

**Pull request merged**